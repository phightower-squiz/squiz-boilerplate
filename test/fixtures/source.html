<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" /><![endif]-->
        <title>{{test}}</title>
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        <!-- import:sass test/fixtures/styles/test.scss [<link rel="stylesheet" href="styles/test.css" media="screen" />] -->
        <!-- import:sass test/fixtures/styles/test.scss [<link rel="stylesheet" href="styles/test.css" media="(min-width: 37.5em)" />] -->
        <!-- import:sassMQ test/fixtures/styles/test.scss [<link rel="stylesheet" href="{{test}}/styles/test.css" media="(min-width: 37.5em)" />] -->

    </head>
    
    <body class="no-js">
    
    <div id="page-wrapper">
    <!--noindex-->

        <div class="header" role="banner">
            <!-- import:content test/fixtures/content.html -->
        </div><!-- /.header -->
    <!--endnoindex-->
            <!-- start main-content -->
            <div id="main" role="main">

            </div><!-- /#main -->
            <!-- end main-content -->
    <!--noindex-->
        <div class="footer" role="contentinfo">

        </div><!-- /.footer -->
    <!--endnoindex-->
    </div><!-- /#page-wrapper -->

    <!-- import:js test/fixtures/js/example.js -->
    <!-- import:js test/fixtures/js/*.js -->

    </body>
</html>