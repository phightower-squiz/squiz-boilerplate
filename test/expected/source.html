<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8" />
        <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" /><![endif]-->
        <title>replaced</title>
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

<link rel="stylesheet" href="styles/test.css" media="screen" />
<link rel="stylesheet" href="styles/test.css" media="(min-width: 37.5em)" />
<link rel="stylesheet" href="replaced/styles/test.css"/>

    </head>
    
    <body class="no-js">
    
    <div id="page-wrapper">
    <!--noindex-->

        <div class="header" role="banner">
<div class="content">This is some content</div>
        </div><!-- /.header -->
    <!--endnoindex-->
            <!-- start main-content -->
            <div id="main" role="main">

            </div><!-- /#main -->
            <!-- end main-content -->
    <!--noindex-->
        <div class="footer" role="contentinfo">

        </div><!-- /.footer -->
    <!--endnoindex-->
    </div><!-- /#page-wrapper -->

<script src="test/fixtures/js/example.js"></script>
<script src="test/fixtures/js/example.js"></script>

    </body>
</html>